automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/library/PublishSpec.scala

```diff
@@ -117,4 +140,17 @@ class PublishSpec extends FreeSpec with WebBrowserSpec with UserFixtures with Wo
       }
     }
   }
+
+  "DUOS autocomplete" - {
+    "should give multiple results for a partial word" in withWebDriver { implicit driver =>
+      val user = UserPool.chooseAnyUser
+      // there is no need for an auth token for this test, except that the api wrapper expects one
+      implicit val authToken: AuthToken = user.makeAuthToken()
+      val partialTextQuery = "dis"
+      val result = api.library.duosAutocomplete(partialTextQuery)
+      val resultCount = partialTextQuery.r.findAllMatchIn(result).length
+      println(resultCount)
+      (resultCount > 5) shouldBe true
```

Comment:
`resultCount should be > 5` should work and will give a better message on failure.

Reply:
👍